README.rst

```diff
@@ -6,7 +6,7 @@ Sysl
 .. image:: https://img.shields.io/appveyor/ci/anz-bank/sysl.svg?label=Windows%20build%20%40%20Appveyor%20CI
    :target: https://ci.appveyor.com/project/anz-bank/sysl
 
-Sysl (pronounced "sizzle") is a system specification language. Using Sysl, you
+Sysl (pronounce "sizzle") is a system specification language. Using Sysl, you
```

Comment:
The original is correct.

Reply:
Sorry, just wanted to trigger another build.... will fix again. Aren't you meant to be on holidays?

Reply:
I am on holidays, and doing what I enjoy! 🤓

Reply:
Fix is in. I wish I had something a bit more interesting for you to enjoy then....